### PR TITLE
NUX Signup: add a skip button to the site information form

### PR DESCRIPTION
**Pull request by southp**
#### Changes proposed in this Pull Request

This PR adds a "skip it" button when the `skipBusinessInformation` a/b test is set as `show`:
image

#### Testing instructions

1. Set `skipBusinessInformation` as `show`
1. Visit `/start/onboarding` and select "Business" at the site type step.
1. In the subsequent site information step, there will be a "Skip this" button. 
1. Hit the button, check the signup state by running `getState().signup`, and you should find that each dependency and progress data is filled by an empty string.
1. Set `skipBusinessInformation` as `hide` and do the above again. This time there won't be a "Skip this" button.

### CODE REVIEW

**Comment by southp** [comment]
Functional-wise it is working, but I'd love some CSS helps. The current way is not responsive at all :/

**Comment by adamziel** [comment]
I'm not sure how @shaunandrews envisions it, but for me the following way is fine:

zrzut ekranu 2019-01-30 o 14 16 03

Whis would mean that we need to add a media query to `.site-information__skip-button` like `@include breakpoint( '<660px' ) { }`, and then inside that query you'd need to add some additional CSS rules like:

```css
    position: relative;
    margin: 0 auto;
    left: 0;
    top: 0;
```

To reset the absolute positioning that every `button` in this area has, and also move it to the center `margin: 0 auto;`. I'm not pushing any commits on purpose - feel free to play with that and ping me if you have any troubles.

**Comment by adamziel** [comment]
I forgot direct ping: @southp

**Comment by fditrapani** [comment]
Thanks for putting this test together. From a copy perspective we can drop "this" so that it only says Skip cc @ranh 

@adamziel I like your suggestion for the mobile view.

**Comment by ranh** [comment]
> Thanks for putting this test together. From a copy perspective we can drop "this" so that it only says Skip cc @ranh

If we want a short and very clear action description, I agree that just "Skip" is enough.

If we want to try something more informative / detailed / helpful, this is what I suggested previously:

> Don’t need help getting started? Skip setup

**Comment by southp** [comment]
@ranh 
Here is the shorter version in action:
image

and the longer version:
image

How do you feel? I've committed the code for the shorter version but I'm happy to revise.

@adamziel Thanks for the tip! I tried to implement your suggestion but it ends up that I have to replace `bottom: 3px` with `top: 29px`, since setting the skip button as relative positioned would expand the wrapping DOM element, causing the continue button to be moved around with it. Do you see any problem with it?

**Comment by ranh** [comment]
Personally I prefer the longer version for the added context. 

The area around the form field feels cluttered, especially on mobile. The title and field label are redundant, we should probably lose one of them.

**Comment by fditrapani** [comment]
> Don’t need help getting started? Skip setup

This seems misleading to me. Are we looking to skip the individual step or the entire flow?

**Comment by ranh** [comment]
> Are we looking to skip the individual step or the entire flow?

Good question. On second (first?) thought, this language really only works on the first step. Otherwise it's not clear as you said if "don't need help" and "skip setup" means that we would also ignore the choices you made so far. 

Just "skip" on it's own can be unclear too though.

What are we trying to say here? Why would customers choose to skip this step? 

Perhaps something like "Leave blank for now" or "Fill this in later" might work better as a label. 

Also, I'm not sure if it's more or less usesable than a link, but we can also add some static microcopy near the field: "Leave blank to skip for now", or otherwise indicate the field as "Optional".

File: client/signup/steps/site-information/index.jsx [hunk 1]
```diff
@@ -112,11 +124,22 @@ export class SiteInformation extends Component {
...
+		const { translate } = this.props;
+
+		return (
+			<>
+				<Button primary type="submit" onClick={ this.handleSubmit }>
+					{ translate( 'Continue' ) }
+				</Button>
+				{ abtest( 'skipBusinessInformation' ) === 'show' && (
+					<Button className="site-information__skip-button" borderless onClick={ this.handleSkip }>
+						{ translate( 'Skip' ) }
```
Comment (taggon) [inline client/signup/steps/site-information/index.jsx]: It's a minor thing. Can we provide context for this simple string, so translators can recognize what the button means?

**Pull request merged**